schema/deploy/shipping/views.sql

```diff
@@ -2310,7 +2310,8 @@ create or replace view shipping.scan_enrollments_v1 as
         scan_study_arm,
         priority_code,
         puma,
-        neighborhood_district
+        neighborhood_district,
+        sample is not null as kit_received
```

Comment:
I'm not sure using `sample is not null` will capture all kits received. The `sample` field is `sample.identifier` so it requires the lab to assign a sample barcode. If a sample cannot be tested, it will not have a `sample.identifier` and will not show as kit received. 

I think `sample is not null OR never_tested` should include all kits?

Reply:
Ah, good point! I will update using your suggestion.

Reply:
Fixed up in the commit. I modified your example slightly to set null results for `never_tested` to be false here